importer/Monument.py

```diff
@@ -487,8 +487,26 @@ def add_to_report(self, key_name, raw_data):
         in the report,
         add it to the report automatically.
         Add direct URL to item in WLM API.
-        """
-        self.problem_report[key_name] = raw_data
+        Optionally, assign a Property ID that the data
+        should have been used as a value for.
+
+        :param key_name: name of the field containing
+                         the problematic data, e.g. the header of the column
+        :type key_name: string
+        :param raw_data: the data that we failed to process
+        :type raw_data: string
+        :param prop_name: name of the property,
+                          as stated in the props library file
+        :type prop_name: string
+        """
+        prop = None
+        if prop_name:
+            try:
```

Comment:
Instead of the try/except you could use `prop = self.props.get(prop_name)` which would return `None` if not found.